<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPerson</class>
 <widget class="QDialog" name="AddPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>241</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>493</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>493</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>اضافه شخص</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="Egypt"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>476</width>
     <height>225</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>الاسم بالكامل (*)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_personName">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="locale">
             <locale language="Arabic" country="Egypt"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>50</number>
            </property>
            <property name="placeholderText">
             <string>الاول</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_personName_2">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="locale">
             <locale language="Arabic" country="Egypt"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>50</number>
            </property>
            <property name="placeholderText">
             <string>الاوسط</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_personName_3">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="locale">
             <locale language="Arabic" country="Egypt"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>50</number>
            </property>
            <property name="placeholderText">
             <string>الاخير</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>الجنس (*)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_gender">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="locale">
           <locale language="Arabic" country="Egypt"/>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <item>
           <property name="text">
            <string>ذكر</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>انثي</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>محل الميلاد</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_birthPlace">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="locale">
           <locale language="Arabic" country="Egypt"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>تاريخ الميلاد</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_birthDate">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::DaySection</enum>
          </property>
          <property name="displayFormat">
           <string>d/M/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
          <property name="date">
           <date>
            <year>1990</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>الرقم القومي</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_natId">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="locale">
           <locale language="Arabic" country="Egypt"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>14</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>الشركه</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_company">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="locale">
           <locale language="Arabic" country="Egypt"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>الوظيفه</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>كاهن</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_jobTitle">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="locale">
             <locale language="Arabic" country="Egypt"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>المؤهل الدراسي</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_education">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="locale">
           <locale language="Arabic" country="Egypt"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>رقم الهاتف</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_mobId">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="locale">
         <locale language="Arabic" country="Egypt"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_addPerson">
        <property name="text">
         <string>اضافه</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_mainWinReturn">
        <property name="text">
         <string>رجوع</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
